src/All-Movies/All-Movies.css

```diff
@@ -0,0 +1,11 @@
+.movies-container {
+        display: grid;
+        grid-template-columns: 300px 300px 300px 300px 300px 300px 300px 300px;
+        grid-gap: 10px;
+        background-color: #444;
+        border-radius: 5px;
+        padding: 20px;
+        font-size: 150%;
+        height: auto;
+        width: 95vw;
+}
```

Comment:
I'm really bad at grid so I would love to get some advice on stuff here, this is just a rough draft so everything is readable

Reply:
I think it looks nice! I like that there are some dynamic values. I think there is shorthand for grid that I actually can recall in regards to the column/row/gap. I left one suggestion on how to dry it up.